app/extensions/brave/locales/en-US/menu.properties

```diff
@@ -13,6 +13,7 @@ openLocation=Open Location…
 openSearch=Search for "{{selectedVariable}}"
 importFrom=Import from…
 closeWindow=Close Window
+closeTabPage=Close tab page
```

Comment:
I think it's OK as-is; it matches the English and also consistent format with `closeWindow`